packages/devtools_app/integration_test/test_infra/run/_io_utils.dart

```diff
@@ -59,4 +63,22 @@ mixin IOMixin {
   static void _defaultPrintCallback(String line, {String tag = ''}) {
     print(tag.isNotEmpty ? '$tag - $line' : line);
   }
+
```

Comment:
this code was moved out of the `_test_app_driver.dart` to be generally applicable to any process.